src/omero_upload/cli.py

```diff
@@ -51,22 +54,53 @@ def _complete(self, text, line, begidx, endidx):
 
     def _configure(self, parser):
         parser.add_argument("file", nargs="+")
+        parser.add_argument(
+            "-m", "--mimetype", help="Mimetype of the file")
+        parser.add_argument(
+            "--data-dir", type=str,
+            help="Path to the OMERO data directory. If passed will try to"
```

Comment:
```suggestion
            help="Path to the OMERO data directory. If passed will try to "
```
Space after 'to', otherwise gets printed as '...will try toin-place...' .